przelewanka.ml

```diff
@@ -0,0 +1,73 @@
+(* Autor: Antoni Żewierżejew *)
+(* Reviewer: Jakub Nowak     *)
+
+(* zwraca NWD liczb a i b *)
+let rec nwd a b =
+    if a < b then nwd b a else
+    if b = 0 then a else
+    nwd b (a mod b)
+
+open Array
```

Comment:
Jak już piszesz `open Array` to nie powinieneś używać `Array.` (np. `Array.copy`). Albo nie pisać `open Array` w ogóle. To już jak uważasz, ale lepiej konsekwentnie.